frontend/packages/ceph-storage-plugin/src/components/ocs-install/existing-cluster-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Trans, useTranslation } from 'react-i18next';
+import { match as RouteMatch } from 'react-router';
+import { Button } from '@patternfly/react-core';
+import { Modal } from '@console/shared/src/components/modal';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src';
+import { history, resourcePathFromModel } from '@console/internal/components/utils';
+import { ListKind, referenceForModel } from '@console/internal/module/k8s';
+import { StorageClusterKind } from '../../types';
+import { OCSServiceModel } from '../../models';
+
+export const ExistingClusterModal: React.FC<ExistingClusterModalProps> = ({
+  match,
+  storageCluster,
+}) => {
+  const {
+    params: { ns, appName },
+  } = match;
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState(true);
+
+  const clusterName = storageCluster.items.find((sc) => sc.status.phase !== 'Ignored').metadata
+    .name;
+
+  const storageClusterPath = `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
+    OCSServiceModel,
+  )}/${clusterName}`;
+
+  const onConfirm = () => {
+    setIsOpen(false);
+    history.push(resourcePathFromModel(ClusterServiceVersionModel, appName, ns));
+  };
+
+  const onCancel = () => {
+    setIsOpen(false);
+    history.push(storageClusterPath);
+  };
+
+  return (
+    <Modal
+      title={t('ceph-storage-plugin~Storage Cluster exists')}
+      titleIconVariant="warning"
+      isOpen={isOpen}
+      variant="small"
+      onClose={() => history.goBack()}
+      isFullScreen={false}
+      actions={[
+        <Button key="confirm" variant="primary" onClick={onConfirm}>
```

Comment:
aria-label for the button

Reply:
There is text in the button so it will be used.
Adding aria-label will be redundant here, since both the text will be remain same. aria-label is good to add where we dont have clue of what button is doing like <button class="burger"></button> or <button>+</button>

Reply:
makes sense. thanks